docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+  alpaca-lora:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    image: alpaca-lora
+    command: generate.py
+    restart: unless-stopped
+    environment:
+      - BASE_MODEL=decapoda-research/llama-7b-hf  # Hugging face model to use
+      - LORA_WEIGHTS=tloen/alpaca-lora-7b
+      - LOAD_8BIT=0
+    volumes:
+      - /tmp:/root/.cache  # Location downloaded weights will be stored
```

Comment:
Probably a better place to just us a docker volume, since we don't need the bind mounting here.

Reply:
What are your thoughts on instead of copying the files to the workspace, we mount the alpaca-lora dir ?  This way instead of having to rebuild/upload to container we can modify generate.py or cleaned_data locally.

Reply:
Well then you cannot run the container on its own, can the image couldn't be shared w/o the repo. This is how VSCode DevContainers work, but this would cause more problems than in solves in this case.